extensions/drafts/package.json

```diff
@@ -115,6 +118,21 @@
           "required": true
         }
       ]
+    },
+    {
+      "name": "quick-capture",
+      "title": "Quick Capture Draft",
```

Comment:
I would suggest to remove the "Quick" from the command name.

Reply:
Thanks!  Appreciate and agree with the feedback.
Updated confirmation message, and removed "Quick" from command name (and all references to it) in f66b533f